examples/v1/taskruns/beta/stepaction-params.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: tekton.dev/v1alpha1
```

Comment:
I might be missing some of the context here - the original reason that we are working on `StepAction` is that we do not want to roll it out in `v1`? is this correct? 

I wonder if this change implies that when it gets into stable it'll be in `v1`?

Reply:
That's right. When StepActions gets to stable, it will be in `v1`. We just promoted it from v1alpha1 to v1beta1. 

The reason it's in this folder is because the Taskrun that calls the stepaction is `v1`. In the TEP, we mentioned that we will promote both the feature flag and the CRD level at the same time.